pkgs/by-name/ir/irccat/package.nix

```diff
@@ -2,26 +2,37 @@
   lib,
   buildGoModule,
   fetchFromGitHub,
+  fetchpatch,
 }:
 
 buildGoModule rec {
   pname = "irccat";
-  version = "0.4.8";
+  version = "0.4.11";
 
   src = fetchFromGitHub {
     owner = "irccloud";
     repo = "irccat";
     rev = "v${version}";
-    hash = "sha256-fr5x1usviJPbc4t5SpIVgV9Q6071XG8eYtyeyraddts=";
+    hash = "sha256-DwJTwpkb2/YHocUHhF0DwxEzDW6DLOxADda/Ed5Xg7c=";
   };
 
-  vendorHash = "sha256-IRXyM000ZDiLPHX20lXlx00tkCzBe5PqvdgXAvm0EAw=";
+  patches = [
+    # predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
+    # https://github.com/irccloud/irccat/pull/38
+    (fetchpatch {
+      url = "https://github.com/irccloud/irccat/pull/38/commits/6bbe6e5762e93080f916a87a93aaeb90ec5fe12c.patch";
```

Comment:
Not a stable URL. Needs to be vendored as documented in pkgs/README.md.

Reply:
Done!